package.json

```diff
@@ -4,16 +4,18 @@
   "description": "Twilio Video SDK Quick Start for JavaScript",
   "main": "index.js",
   "scripts": {
-    "clean": "npm run clean:quickstart && npm run clean:examples",
-    "clean:quickstart": "rimraf quickstart/public/index.js",
-    "clean:examples": "npm run clean:examples:mediadevices && npm run clean:examples:localvideosnapshot",
-    "clean:examples:mediadevices": "rimraf examples/mediadevices/public/index.js examples/mediadevices/public/helpers.js",
-    "clean:examples:localvideosnapshot": "rimraf examples/localvideosnapshot/public/index.js examples/localvideosnapshot/public/helpers.js",
-    "build": "npm run build:quickstart && npm run build:examples",
-    "build:examples": "npm run build:examples:mediadevices && npm run build:examples:localvideosnapshot",
-    "build:examples:mediadevices": "copyfiles -f examples/mediadevices/src/helpers.js examples/mediadevices/public && browserify examples/mediadevices/src/index.js > examples/mediadevices/public/index.js",
+    "build": "npm-run-all build:*",
+    "build:examples": "npm-run-all build:examples:*",
+    "build:examples:localvideofilter": "copyfiles -f examples/localvideofilter/src/helpers.js examples/localvideofilter/public && browserify examples/localvideofilter/src/index.js > examples/localvideofilter/public/index.js",
     "build:examples:localvideosnapshot": "copyfiles -f examples/localvideosnapshot/src/helpers.js examples/localvideosnapshot/public && browserify examples/localvideosnapshot/src/index.js > examples/localvideosnapshot/public/index.js",
+    "build:examples:mediadevices": "copyfiles -f examples/mediadevices/src/helpers.js examples/mediadevices/public && browserify examples/mediadevices/src/index.js > examples/mediadevices/public/index.js",
     "build:quickstart": "browserify quickstart/src/index.js > quickstart/public/index.js",
+    "clean": "npm-run-all clean:*",
+    "clean:examples": "npm-run-all clean:examples:*",
+    "clean:examples:localvideofilter": "rimraf examples/localvideofilter/public/index.js examples/localvideofilter/public/helpers.js",
+    "clean:examples:localvideosnapshot": "rimraf examples/localvideosnapshot/public/index.js examples/localvideosnapshot/public/helpers.js",
+    "clean:examples:mediadevices": "rimraf examples/mediadevices/public/index.js examples/mediadevices/public/helpers.js",
```

Comment:
@markandrus I'll include this as part of the refactoring PR that will include the changes you suggested in the other examples.